github-dark.css

```diff
@@ -1956,7 +1956,7 @@
   .state-indicator.renamed, .discussion-topic .branch-status.status-pending,
   .compare-pr-placeholder, .compare-cutoff, .diff-cutoff, .flash.flash-warn,
   .flash-global.flash-warn, .markdown-body li.added.moved, .repo-private-label,
-  .gist-secret-label, .label-private, .stale-files-tab {
+  .gist-secret-label, .label-private, .RecentBranches, .stale-files-tab {
```

Comment:
Are you sure about this class name? I don't think I've ever seen a class name (in GitHub) that starts with a capital letter or camel cased... I would have expected to see something like `.recent-branches`.